pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>org.apereo.uportal</groupId>
       <artifactId>uportal-app-framework</artifactId>
-      <version>6.1.1-SNAPSHOT</version>
+      <version>7.0.0</version>
```

Comment:
👍 . Was Widget Creator deliberately using a `SNAPSHOT` so as to be another canary in the mine about breaking framework changes? Worth using `7.0.1-SNAPSHOT` here for that continued canary?

Reply:
Not sure why it was using a `SNAPSHOT`. I will defer to whoever has an opinion about this.

Reply:
:+1: to using SNAPSHOT. This way we get the latest running on the test tier. I think I changed it to SNAPSHOT the last time I was working on it.

Reply:
Also, since we do manual interactive releases of this, there's no real danger of keeping this on SNAPSHOT

Reply:
Done